### PR TITLE
Change iter.next() to next(iter)

### LINKED ISSUE #9086: Python 3: use next(iter) instead of iter.next()
Hi,

Yet another syntax issue to resolve:

### Is your feature request related to a problem? Please describe.
In Python 2.x, it was possible to go through iterators via iter.next() method. This was made simple in Python 2.6 with the built-in next() function, and this is the only method for built-in iteration control in python 3.

### Describe the solution you'd like
As NVDA uses Python 2.7, it will be able to work with next(iter), making it compatible with Python 3. This will affect the following modules:

* GUI/settings dialogs: retrieving options for one or two settings.
* Event handler: handling next chain of events.
* Say all handler: say all generator.

### Describe alternatives you've considered
None

### Additional context
None

### PR DESCRIPTION
**Pull request by jakubl7545**
### Link to issue number:
closes #9086
### Summary of the issue:
See issue description.
### Description of how this pull request fixes the issue:
As suggested I've changed iter.next() to next(iter).
### Testing performed:
Only ran NVDA from source, because I'm not sure whether it needs any special testing scenerios.
### Known issues with pull request:
None
### Change log entry:
None
cc @josephsl

### CODE REVIEW

**Comment by josephsl** [comment]
CC @LeonardDer - let's defer this until speech refactor lands.

File: source/eventHandler.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def __init__(self, eventName, obj, kwargs):
 		self.kwargs = kwargs
 		self._gen = self.gen(eventName, obj)
 		try:
-			self.next()
+			next(self)
```
Comment (leonardder) [inline source/eventHandler.py]: I'm afraid this is subject to serious breakage as it is now. You're calling next(self) instead of self.next(). However, next is a method on self. Here, you're not dealing with a real iterator, but with a class that only inherits from object.

I'd like to suggest to revert these changes to eventHandler altogether.

File: source/sayAllHandler.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def readObjects(obj):
 	_startGenerator(readObjectsHelper_generator(obj))
```
Comment (leonardder) [inline source/sayAllHandler.py]: All changes to this file are going to conflict with #7599. Please revert these as well. They will be addressed during the Python 3 transition.

**Review by LeonarddeR — CHANGES_REQUESTED** [review]

File: source/sayAllHandler.py (hunk 2 above)
Comment (jakubl7545) [reply source/sayAllHandler.py]: I've reverted the changes.

**Review by LeonarddeR — APPROVED** [review]
@feerrenrut: This is now a one liner, so this can be merged into master I think.

I'm sorry that I had to request to strip down the pull request this way.

**Comment by jakubl7545** [comment]
Wouldn't be better to wait for merging speech refactor and then check if sayAllHandler still use iter.next()?

**Comment by feerrenrut** [comment]
> Wouldn't be better to wait for merging speech refactor and then check if sayAllHandler still use iter.next()?

Yes, I think this makes sense. You will need to change the target of this PR to be the threshold branch, which is where we will be doing the python 3 work.

**Comment by leonardder** [comment]
I believe that with dropping speechCompat, there is no longer a call of next like this in sayAllHandler.

**Comment by jakubl7545** [comment]
I changed target branch to threshold and i found that sayAllHandler uses "reader.next()" in line 40 (readObjects function).
@leonardder should this line be modified to next(reader)?

**Comment by leonardder** [comment]
Nope, please leave this as is. next is a function on _ObjectsReader.

**Comment by jakubl7545** [comment]
So i think it can be merged as it is.

**Review by feerrenrut — APPROVED** [review]
